contrib/win32/win32compat/pwd.c

```diff
@@ -236,9 +236,11 @@ get_passwd(const wchar_t * user_utf16, PSID sid)
 		goto cleanup;
 	}
 
```

Comment:
Please add steps to verify the new functionality.

Reply:
To verify the functionality set the following in the ssh_config
AuthorizedKeysCommandUser=system

I'm looking at how to extend the regress\pesterTests\AuthorizedKeysCommand.Tests.ps1 test to allow the AuthorizedKeysCommandUser to be specified instead of always using $ssouser. So then there would be 2 runs of the tests, one with the existing $ssouser and the other as system.

Reply:
I'm not sure why the following test failed in the appveyor build test.
```Run transfer.sh [9 of 71] 03/18/2021 16:39:38
transfer.sh FAILED 03/18/2021 16:42:05
```

I ran the test suite against the combined changes and also this specific change by using the same commands run to test only this change in this [test run](https://ci.appveyor.com/project/PowerShell/openssh-portable/builds/38293371) and they succeeded.

```
Run transfer.sh [9 of 71] 03/19/2021 15:14:05
transfer.sh PASSED 03/19/2021 15:14:18
...
{
    "StartTime":  "3/19/2021 3:12:37 PM",
    "EndTime":  "3/19/2021 4:06:50 PM",
    "TotalExecutionTime":  "00:54:12",
    "TotalBashTests":  71,
    "TotalBashTestsPassed":  64,
    "TotalBashTestsFailed":  0,
    "TotalBashTestsSkipped":  7,
    "FailedBashTests":  "",
    "SkippedBashTests":  "agent.sh, key-options.sh, cert-hostkey.sh, cert-userkey.sh, forward-control.sh, integrity.sh, krl.sh",
    "BashTestSummaryFile":  "C:\\projects\\openssh-portable\\regress\\bash_tests_summary.txt",
    "BashTestLogFile":  "C:\\projects\\openssh-portable\\regress\\bash_tests_output.log"
}
```

Reply:
From log:
```
Run transfer.sh [9 of 71] 03/18/2021 16:39:38
generating key type ssh-ed25519
generating key type ssh-rsa
generating key type ssh-dss
generating key type ecdsa-sha2-nistp256
generating key type ecdsa-sha2-nistp384
generating key type ecdsa-sha2-nistp521
dd-size 10
ssh cat /cygdrive/c/projects/openssh-portable/regress/temp_test/data failed
cmp: /cygdrive/c/projects/openssh-portable/regress/temp_test/copy: No such file or directory
corrupted copy
dd-size 100
dd-size 1k
dd-size 32k
dd-size 64k
dd-size 128k
dd-size 256k
failed transfer data
transfer.sh FAILED 03/18/2021 16:42:05
```

Reply:
@bkatyl, I did minor changes and resolved the merge conflicts in pwd.c. My intention is  to restart the build as it appears transfer.sh failed randomly.

Hope you don't mind.

Reply:
That's great, thank you. It looks like all the tests now pass. :)